--- a/Makefile.am
+++ b/Makefile.am
@@ -1,11 +1,11 @@
 ACLOCAL_AMFLAGS = -I m4
-SUBDIRS = src include tests fortran
+SUBDIRS = src include tests fortran 
 
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = qd.pc
 
-BUILT_SOURCES = docs/qd.pdf
-dist_doc_DATA = README docs/qd.pdf
+BUILT_SOURCES = 
+dist_doc_DATA = README 
 dist_noinst_DATA = docs/Makefile \
   docs/qd.tex docs/qd.bib \
   docs/nine-two-sum.eps docs/nine-two-sum.fig \
@@ -23,10 +23,10 @@
   docs/two-prod.eps docs/two-prod.fig \
   docs/two-sum.eps docs/two-sum.fig
 
-doc: docs/qd.pdf
+# doc: docs/qd.pdf
 
-docs/qd.pdf:
-	$(MAKE) -C docs qd.pdf
+#docs/qd.pdf:
+#	$(MAKE) -C docs qd.pdf
 
 cpp-demo:
 	$(MAKE) -C tests demo
